core/feature_flag_list.py

```diff
@@ -49,6 +49,7 @@ class FeatureNames(enum.Enum):
     IS_IMPROVEMENTS_TAB_ENABLED = 'is_improvements_tab_enabled'
     LEARNER_GROUPS_ARE_ENABLED = 'learner_groups_are_enabled'
     NEW_LESSON_PLAYER = 'new_lesson_player'
+    UNDO_TRANSLATION_STATUS = 'undo_translation_status'
```

Comment:
Can you name this something that makes it clear it's for the CD?

Also it's not clear what "undoing status" means, status is not an action that can be undone. Perhaps CD_ALLOW_UNDOING_TRANSLATION_REVIEW or something like that.

Reply:
Done, `cd_allow_undoing_translation_review` SGTM, Thanks!!